include/kernel/pit.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: CGL-KFS
+// SPDX-License-Identifier: BSD-3-Clause
+
+/* include/kernel/pit.h
+ *
+ *
+ *
+ * created: 2022/12/14 - xlmod <glafond-@student.42.fr>
+ * updated: 2022/12/15 - xlmod <glafond-@student.42.fr>
+ */
+
+#ifndef PIT_H
+#define PIT_H
+
+#include <stdint.h>
+
+#define PIT_TIMEMS ((pit_counter * 1000) / pit_freq)
+
+extern uint32_t pit_freq;
+extern uint32_t pit_counter;
```

Comment:
I think this variable should be marked as volatile, because it can be changed asynchronously by an interrupt while polled from a function e.g. `sleep`. Marking it as volatile would avoid the compiler to make assumptions and loop forever --> I'm not 100% sure about it, let's check.

Reply:
done